packages/react/tsconfig.json

```diff
@@ -2,11 +2,13 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "isolatedModules": false,
-    "outDir": "./dist"
+    "outDir": "./dist",
+    "types": ["jest"]
+
   },
   "include": [
     "./src/**/*.ts"
-  ],
+, "src/QueeryKeeper.spec.tsx"  ],
```

Comment:
what does it mean?

Reply:
was generated automatically after `yarn test` command